ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/ParsePayload.java

```diff
@@ -145,6 +143,9 @@ protected PubsubMessage processElement(PubsubMessage message)
 
     addAttributesFromPayload(attributes, json);
 
+    // Write messages that are affected by a specific bug to error output.
+    MessageScrubber.writeToErrors(attributes, json);
```

Comment:
Moving this down here, both Glean and legacy telemetry data should get handled correctly.